src/CMake/FindVTKm.cmake

```diff
@@ -67,19 +68,17 @@ IF (DEFINED VISIT_VTKH_DIR)
        endif()
    endfunction()
 
-   get_target_property(VTKH_INT_LL vtkh INTERFACE_LINK_LIBRARIES)
+   get_target_property(VTKM_INT_LL vtkm_filter INTERFACE_LINK_LIBRARIES)
    set(addl_ll)
-   foreach(vtkhll ${VTKH_INT_LL})
-       get_lib_loc_and_install(${vtkhll})
-       get_target_property(VTKH_LL_DEP ${vtkhll} INTERFACE_LINK_LIBRARIES)
-       if(VTKH_LL_DEP)
-           foreach(ll_dep ${VTKH_LL_DEP})
-               string(SUBSTRING "${ll_dep}" 0 4 ll_dep_prefix)
-               # only process libraries that start with vtkh or vtkm
-               if ("${ll_dep_prefix}" STREQUAL "vtkh" OR
-                   "${ll_dep_prefix}" STREQUAL "vtkm")
+   foreach(vtkmll ${VTKM_INT_LL})
+       get_lib_loc_and_install(${vtkmll})
+       get_target_property(VTKM_LL_DEP ${vtkmll} INTERFACE_LINK_LIBRARIES)
+       if(VTKM_LL_DEP)
+           foreach(ll_dep ${VTKM_LL_DEP})
+               # only process libraries that start with vtkm
+               if ("${ll_dep_prefix}" STREQUAL "vtkm")
                    # don't process duplicates
-                   if (NOT ${ll_dep} IN_LIST VTKH_INT_LL AND
+                   if (NOT ${ll_dep} IN_LIST VTKM_INT_LL AND
```

Comment:
To answer your question regarding if some of this logic is still necessary for vtkm. I would put some message statements in the get_lib_loc_and_install function to see the names (if any) of the libs being passed to THIRD_PARTY_INSTALL_LIBRARY.

Reply:
I added a message statement and none of the VTKm libraries are passed to THIRD_PARTY_INSTALL_LIBRARY. This is because they are all static libraries. I left the code in and added a comment to that affect if we ever switch to shared libraries.